packages/epo-widget-lib/src/widgets/ColorTool/FilterImage/FilterImage.tsx

```diff
@@ -34,13 +34,17 @@ const FilterImage: FunctionComponent<FilterImageProps> = ({
     const ctx = canvasRef.current?.getContext("2d");
 
     if (ctx) {
+      const safeColor =
+        CSS.supports && CSS.supports("background-color", color)
+          ? color
+          : "transparent";
```

Comment:
make sure that the color being applied is a real color and transparent if it's not